app/views/analytics/system_activity_logs.html.erb

```diff
@@ -15,7 +15,7 @@
   <tbody>
     <% @logs.each do |log| %>
       <tr>
-        <td><%= log.user.fullname %></td>
+        <td><%= log.user.present? ? log.user.fullname : "Unknown User" %></td>
```

Comment:
also
```
<td><%= log&.user&.fullname || "Unknown User" %></td>
```

Reply:
Well, `log` can't be null, so `<%= log.user&.fullname || "Unknown User" %>` would also be acceptable ?

Reply:
even better. 👍